nautilus_trader/adapters/interactive_brokers/config.py

```diff
@@ -170,6 +170,9 @@ class InteractiveBrokersDataClientConfig(LiveDataClientConfig, frozen=True):
     market_data_type : bool, default REALTIME
         Set which IBMarketDataTypeEnum to be used by InteractiveBrokersClient.
         Configure `IBMarketDataTypeEnum.DELAYED_FROZEN` to use with account without data subscription.
+    quote_ticks_ignore_size : bool
```

Comment:
A more suitable name for this flag could be `ignore_quote_tick_size_updates`, with the following description:

"If set to True, the QuoteTick subscription will exclude ticks where only the size has changed but not the price. This can help reduce the volume of tick data. When set to False (the default), QuoteTick updates will include all updates, including those where only the size has changed."

Reply:
I have made changes.
Thank you for your suggestions.